sequencer/src/block/full_payload/ns_table.rs

```diff
@@ -0,0 +1,229 @@
+//! Types related to a namespace table.
+//!
+//! All code that needs to know the binary format of a namespace table is
+//! restricted to this file.
```

Comment:
Would be nice to add more comments following this helpful document: https://www.notion.so/espressosys/Block-payload-format-with-namespace-transaction-tables-81425c5bd3ba495bafc3198ce287c487

Reply:
addressed in https://github.com/EspressoSystems/espresso-sequencer/pull/1499/commits/5c9fe51fa9f95bb392d5c8446166068663ef0ac1. Added a detailed spec in rustdoc. Also added namespace payload spec in https://github.com/EspressoSystems/espresso-sequencer/pull/1499/commits/311a7f3353cc24194b401e81df7b03f4893b8a22